### DISCUSSION TITLE
Project 2

### DISCUSSION THREAD

**Discussion by courtcarroll** [cooperrc/computational-mechanics#105] [Q&A]
For project 2 are we using the code that is given and changing it for a different stock or should we use that code to guide us

**Comment by cooperrc** [comment]
Using the code given is great :+1: 

You can add edit/add however you see fit.

**Comment by saewon1** [comment]
I used a modified version of the code that was given

**Comment by ChristianGurrieri** [comment]
I used the code that was given and modified some parts of it.
